File: frontend/src/features/actividades/corregir_reparo_calculo_doc/models/corregir_reparo_calculo_doc.ts
export interface CorregirReparoCalculoDoc {
  id_corregir_reparo_calculo_doc: number;
  id_expediente: number;
  id_usuario_solicitante: number;
  is_subsanar: boolean;
  observaciones: string | null;

  existe_rol_avaluo: string | null;
  rol_avaluo_editado: string | null;
  valor_avaluo_pesos: number | null;

  /**
   * Campos autocompletados desde 5.13 Cálculo y Generación Documento
   * y datos_operacion_propiedad. Vienen del backend con [NotMapped]
   * y son de sólo lectura en la UI.
   */
  solicitante?: string | null;
  observaciones_reparo?: string | null;
  fecha_ingreso?: string | null;
  tipo_propiedad?: string | null;
  tipo_direccion?: string | null;
  direccion?: string | null;
  region?: string | null;
  comuna?: string | null;
  rol_avaluo?: string | null;

  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by: number | null;
  modified_date: string | null;
}

File: backend/Multibanca.Domain.Models/Multibanca/corregir_reparo_calculo_doc.cs
using Common.Domain.Models;
using System;

namespace Multibanca.Domain.Models.Multibanca
{
    public class corregir_reparo_calculo_doc : base_auditoria
    {
        public int id_corregir_reparo_calculo_doc { get; set; }

        public long id_expediente { get; set; }

        public int id_usuario_solicitante { get; set; }

        public bool is_subsanar { get; set; }

        public string? observaciones { get; set; }

        public string? existe_rol_avaluo { get; set; }

        public string? rol_avaluo_editado { get; set; }

        public decimal? valor_avaluo_pesos { get; set; }

        public string? solicitante { get; set; }

        public string? observaciones_reparo { get; set; }

        public DateTime? fecha_ingreso { get; set; }

        public string? tipo_propiedad { get; set; }

        public string? tipo_direccion { get; set; }

        public string? direccion { get; set; }

        public string? region { get; set; }

        public string? comuna { get; set; }

        public string? rol_avaluo { get; set; }
    }
}
